.github/workflows/build-binaries.yml

```diff
@@ -5,6 +5,11 @@ on:
     tags:
       - 'v*'
   workflow_dispatch:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - .github/workflows/build-binaries.yml
```

Comment:
does this mean that the workflow run after PR is made on main branch ?

Reply:
it means that this workflow will run on PRs that change this file (for example this one)